src/tools/annotation/FreehandMouseTool.js

```diff
@@ -110,7 +112,7 @@ export default class FreehandMouseTool extends BaseAnnotationTool {
    * @returns {Boolean}
    */
   pointNearTool(element, data, coords) {
-    const validParameters = data && data.handles;
+    const validParameters = data && data.handles && data.handles.dataHandles;
```

Comment:
Hmm, could we rename the `dataHandles` array to just `points`. This would avoid tongue twisters like `data.handles.dataHandles`.

Reply:
agree, this is actually what I had in mind initially but I saw "dataHandles" used in the code to reference this structure so I re-used it.

Reply:
That's fair, I originally used `dataHandles` as a shorthand for the reference to `data.handles`, but if we have another property under `data.handles` as we do here, we should give it a new name 👍 .

Reply:
Renamed to "points"